### ISSUE TITLE
arm64 not getting pulled

### ISSUE THREAD

**Issue by muhammadtahseen0345** [keymetrics/docker-pm2#100]
We cannot pull arm64 based pm2 image. Flags at the time of pull also not working.
Add arm64 based architecture compatibility.

**Comment by timnolte** [comment]
Yeah, there are no arm64 images built for PM2. Only `linux/amd64`. https://hub.docker.com/r/keymetrics/pm2/tags

**Comment by muhammadtahseen0345** [comment]
I have uploaded my custom image that is arm64 based on docker hub . 
Thanks for the concern. 
 
  On Fri, Sep 8, 2023 at 12:07 AM, Tim ***@***.***> wrote:   


Yeah, there are no arm64 images built for PM2. Only linux/amd64. https://hub.docker.com/r/keymetrics/pm2/tags
